ci/Dockerfile.local

```diff
@@ -3,9 +3,8 @@
 FROM python:3.8-alpine3.15
 
 ENV PYTHONDONTWRITEBYTECODE=1 \
-    # With this environment variable set, the Flask development server will print
-    # debug output and automatically restart in response to code changes.
-    FLASK_ENV=development
+    # https://flask.palletsprojects.com/en/2.2.x/config/?highlight=flask_debug#DEBUG
+    FLASK_DEBUG="true"
```

Comment:
For local development, we should get debug output, and the dev server should restart automatically when code changes are saved.  I made changes last year to make this happen, but it never worked correctly.  More recent versions of Flask changed how to make this happen.  One of the requirements is to set the FLASK_DEBUG variable, as I have here for the local container.  Elsewhere, I pass the "debug" flag when running Flask.  It seems to work this time.

https://flask.palletsprojects.com/en/2.2.x/changes/#version-2-2-3) and ](https://flask.palletsprojects.com/en/2.2.x/config/?highlight=flask_debug#debug-mode